src/main/java/ua/lviv/iot/java/labs/forth/models/Bag.java

```diff
@@ -0,0 +1,35 @@
+package ua.lviv.iot.java.labs.forth.models;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class Bag extends Good {
+  private String pattern;
+  private int capacityInLitres;
+  private float weightInKg;
+  private String brand;
+
+  public Bag(String name, float priceInUAH, Gender forGender, boolean isDiscounted, long id,
+      String origin, String pattern, int capacityInLitres, float weightInKg, String brand) {
+    super(name, priceInUAH, forGender, isDiscounted, id, origin);
+    this.setPattern(pattern);
```

Comment:
why do you call setPattern in constructor instead of initializing it like this this pattern = pattern?
Don't you think it's redundant method call?

Reply:
same for all, except for the last one

Reply:
Done (while testing Manager had a problem with Eclipse wanting to test setters/getters, which was unnecessary, and after resolving that I forgot to change setters usage in constructors to a common usage e.g. this.field = field)